modules/installation-gcp-shared-vpc-config.adoc

```diff
@@ -0,0 +1,77 @@
+// This file is referenced in the following assembly:
+// installing/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-config_{context}"]
+= Sample customized `install-config.yaml` file for shared VPC installation
+There are several configuration parameters which are required to install {product-title} on GCP using a shared VPC. The following is a sample `install-config.yaml` file which demonstrates these fields.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must modify this file with the correct values for your environment and cluster.
+====
+
```

Comment:
I kind of wonder if we could add a ".Example `instlal-config.yaml` file above the actual YAML? Not necessary, but might _look_ better.

Reply:
@bscott-rh If you choose to add the `install-config.yaml` example, maybe we should add or note the `privateDNSZone` as an option there but clearly indicate that it is not required (and seldom used).

Reply:
@barbacbd I think Steven was referring to just adding a sub-heading of "Example `install-config.yaml` file" immediately before the code block.

I think it would be best to keep this module consistent with the sample install yaml modules for the other platforms and installation methods, which don't have the extra line before the code block.